egs2/chime4/asr2/README.md

```diff
@@ -0,0 +1,96 @@
+   <details>
+   <summary>bib info</summary>
+
+   ```
+   @article{chang2023exploration,
+        title={Exploration of Efficient End-to-End ASR using Discretized Input from Self-Supervised Learning},
+        author={Chang, Xuankai and Yan, Brian and Fujita, Yuya and Maekaku, Takashi and Watanabe, Shinji},
+        journal={arXiv preprint arXiv:2305.18108},
+        year={2023}
+   }
+   ```
+   </details>
+
+# E-Branchformer ASR2 Discrete tokens with WavLM_large_Layer21_Kmeans1000_nBPE2000
+
+# RESULTS
+## Environments
+- date: `Mon Sep 11 04:53:07 EDT 2023`
+- python version: `3.10.10 (main, Mar 21 2023, 18:45:11) [GCC 11.2.0]`
+- espnet version: `espnet 202308`
+- pytorch version: `pytorch 1.13.1+cu117`
+- Git hash: `83e687f3b41310a000f4a5b65857734709752bf6`
+  - Commit date: `Tue Aug 15 18:31:02 2023 -0400`
+
+## exp/asr_train_discrete_asr_e_branchformer_e12_mlp1024_linear1024_macaron_lr1e-4_warmup25k_conv1d1_raw_wavlm_large_21_km1000_bpe_rm2000_char_ts_sp
+
+- ASR config: [conf/tuning/train_discrete_asr_e_branchformer_e12_mlp1024_linear1024_macaron_lr1e-4_warmup25k_conv1d1.yaml](conf/tuning/train_discrete_asr_e_branchformer_e12_mlp1024_linear1024_macaron_lr1e-4_warmup25k_conv1d1.yaml)
+- decode config: [conf/decode_asr.yaml](conf/decode_asr.yaml)
+- Model link: [https://huggingface.co/espnet/eason_chime4_asr2_e_branchformer12_conv1d1_raw_wavlm_large_21_km1k_bpe_rm2k_char_ts_sp](https://huggingface.co/espnet/eason_chime4_asr2_e_branchformer12_conv1d1_raw_wavlm_large_21_km1k_bpe_rm2k_char_ts_sp)
+
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_asr_model_valid.acc.ave/dt05_real_beamformit_2mics|1640|27119|89.9|8.6|1.5|0.8|10.9|58.5|
+|decode_asr_asr_model_valid.acc.ave/dt05_real_beamformit_5mics|1640|27119|92.2|6.6|1.2|0.6|8.4|54.3|
+|decode_asr_asr_model_valid.acc.ave/dt05_real_isolated_1ch_track|1640|27119|89.0|9.6|1.4|0.9|11.9|63.5|
+|decode_asr_asr_model_valid.acc.ave/dt05_simu_beamformit_2mics|1640|27120|89.8|8.4|1.7|0.7|10.8|61.8|
+|decode_asr_asr_model_valid.acc.ave/dt05_simu_beamformit_5mics|1640|27120|93.3|5.7|1.0|0.4|7.1|54.9|
+|decode_asr_asr_model_valid.acc.ave/dt05_simu_isolated_1ch_track|1640|27120|84.4|12.8|2.8|0.8|16.5|67.1|
+|decode_asr_asr_model_valid.acc.ave/et05_real_beamformit_2mics|1320|21409|90.5|8.2|1.3|0.6|10.1|66.7|
+|decode_asr_asr_model_valid.acc.ave/et05_real_beamformit_5mics|1320|21409|92.8|6.3|0.9|0.5|7.7|58.0|
+|decode_asr_asr_model_valid.acc.ave/et05_real_isolated_1ch_track|1320|21409|88.3|10.0|1.7|0.8|12.5|71.5|
+|decode_asr_asr_model_valid.acc.ave/et05_simu_beamformit_2mics|1320|21416|89.0|9.2|1.8|0.9|11.8|66.4|
+|decode_asr_asr_model_valid.acc.ave/et05_simu_beamformit_5mics|1320|21416|92.7|6.4|0.9|0.7|8.0|61.0|
+|decode_asr_asr_model_valid.acc.ave/et05_simu_isolated_1ch_track|1320|21416|83.3|13.2|3.5|1.1|17.8|69.4|
+|decode_asr_asr_model_valid.acc.best/dt05_real_beamformit_2mics|1640|27119|89.5|9.2|1.3|0.9|11.4|60.2|
```

Comment:
Just keep the results of `valid.acc.ave`. You can delete `valid.acc.best` results.